Model/articles.model.js

```diff
@@ -36,15 +36,64 @@ exports.updateArticleById = (article_id, inc_votes) => {
     })
 }
 
-exports.selectArticles = () => {
-    
-    return db.query(`SELECT articles.author, articles.title, articles.article_id,articles.topic, articles.created_at, articles.votes, CAST(COUNT(comments.article_id) AS int) as comment_count 
+exports.selectArticles = (sort_by = 'created_at', order, topic) => {
+    const validSortBy = ['created_at', 'author', 'title', 'article_id', 'topic', 'votes', 'comment_count']
+    const validTopics = ['mitch', 'cats', 'paper']
```

Comment:
you should be checking these topics with a query as the topics on the dev-data are different and will break when you try to host